system-addon/lib/ActivityStream.jsm

```diff
@@ -137,6 +138,10 @@ const PREFS_CONFIG = new Map([
   ["section.topstories.collapsed", {
     title: "Collapse the Top Stories section",
     value: false
+  }],
+  ["section.topstories.showDisclaimer", {
```

Comment:
Could you also delete those unneeded prefs at https://github.com/mozilla/activity-stream/pull/3774/commits/268b61176254f1c787938cd977240399386b9501#diff-5f86e62792c72556307c752b5c25c29eL112?